app/services/workflow_find_service.rb

```diff
@@ -9,7 +9,7 @@ def find(tag_attrs)
 
   # find workflows from a collection of [app_name, object_type, [namespace, key, value]]
   def find_by_tag_resources(tag_resources)
-    return [] if tag_resources.empty?
+    return [] if tag_resources.blank?
```

Comment:
@bzwei is this because the caller is passing in a nil? Wouldn't that be caught by the openapi_parser validators

Reply:
yes for api calls. But it is also called by `RequestCreateService`. In a spec test I did not provide tag_resources which blowed up. I think this a safer change.